superset-frontend/src/features/reports/ReportModal/HeaderReportDropdown/index.tsx

```diff
@@ -214,15 +215,17 @@ export default function HeaderReportDropDown({
 
   const textMenu = () =>
     isEmpty(report) ? (
-      <Menu selectable={false} css={onMenuHover}>
+      <Menu selectable={false} {...rest} css={onMenuHover}>
         <Menu.Item onClick={handleShowMenu}>
           {DropdownItemExtension ? (
             <StyledDropdownItemWithIcon>
               <div>{t('Set up an email report')}</div>
               <DropdownItemExtension />
             </StyledDropdownItemWithIcon>
           ) : (
-            t('Set up an email report')
+            <React.Fragment>
```

Comment:
you probably don't need the React.Fragment here since there's a singular div as the child.

Reply:
Sure will remove that.